test/helpers/apollo-client.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  ApolloClient,
+  InMemoryCache,
+  ApolloLink,
+  from,
+  HttpLink,
+  split,
+  NormalizedCacheObject,
+} from "@apollo/client/core"
+import { WebSocketLink } from "@apollo/client/link/ws"
+import { getMainDefinition } from "@apollo/client/utilities"
+import fetch from "cross-fetch"
+import { SubscriptionClient } from "subscriptions-transport-ws"
+import ws from "ws"
+
+export const createApolloClient = (
+  authToken?: string,
+  port = 4002,
+  graphqlPath = "/graphql",
+  graphqlSubscriptionPath = "/graphql",
+): { apolloClient: ApolloClient<NormalizedCacheObject>; disposeClient: () => void } => {
+  const cache = new InMemoryCache()
+
+  const authLink = new ApolloLink((operation, forward) => {
+    operation.setContext(({ headers }: { headers: Record<string, string> }) => ({
+      headers: {
+        authorization: authToken ? `Bearer ${authToken}` : "",
+        ...headers,
+      },
+    }))
+    return forward(operation)
+  })
+
+  const httpLink = new HttpLink({ uri: `http://localhost:${port}${graphqlPath}`, fetch })
+
+  const subscriptionClient = new SubscriptionClient(
+    `ws://localhost:${port}${graphqlSubscriptionPath}`,
+    {
+      reconnect: true,
```

Comment:
does the connection drop frequently during the tests?

Reply:
Not that I know of. I am using the config for the client that we have in the galoy web wallet

Reply:
ok I think this config is not necessary, and maybe better to not have here for the tests. if the connection disconnect for whatever reason, it's better to fail and understand what the issue is, instead of passing because there may be a retry mechanism by activating this

Reply:
I will remove it